pkg/agent/apiserver/apiserver.go

```diff
@@ -199,6 +202,9 @@ func newConfig(aq agentquerier.AgentQuerier,
 		return fmt.Errorf("disconnected from OFSwitch")
 	})
 	serverConfig.LivezChecks = append(serverConfig.LivezChecks, ovsConnCheck)
+	serverConfig.OpenAPIV3Config = genericapiserver.DefaultOpenAPIV3Config(
+		openapi.GetOpenAPIDefinitions,
+		genericopenapi.NewDefinitionNamer(apiserver.Scheme))
```

Comment:
Since `v0.28.0`, the following code has been update, so that the openAPIV3Config cannot be nil.

```go
// v0.28.0 and newer
func (s *GenericAPIServer) getOpenAPIModels(apiPrefix string, apiGroupInfos ...*APIGroupInfo) (managedfields.TypeConverter, error) {
	if s.openAPIV3Config == nil {
		// SSA is GA and requires OpenAPI config to be set
		// to create models.
		return nil, errors.New("OpenAPIV3 config must not be nil")
	}
...

// v0.27.0 and older
func (s *GenericAPIServer) getOpenAPIModels(apiPrefix string, apiGroupInfos ...*APIGroupInfo) (map[string]*spec.Schema, error) {
	if s.openAPIV3Config == nil {
		//!TODO: A future work should add a requirement that
		// OpenAPIV3 config is required. May require some refactoring of tests.
		return nil, nil
	}
```